### ISSUE TITLE
Move getos lib to this repo

### ISSUE THREAD

**Issue by rusher81572** [picluster/picluster#26]
Currently, it exists on https://github.com/aidanharris/getos.git

**Comment by aidanharris** [comment]
The easiest thing to do would probably be for you to fork my repo and then submit a pull request updating the package.json file. In the long term it'd probably be better to publish the module to npm under another name.

**Comment by aidanharris** [comment]
I've changed getos to use an [npm package](https://www.npmjs.com/package/picluster-getos) in 944104770600d961397e70843b796f6743db2115. Hopefully you shouldn't have any more issues installing it now (beyond those caused by npm itself).

The only thing left to do is as follows:

- [ ] Move the repo to the picluster namespace on Github. @rusher81572 needs to create an empty getos repo for me to push to or fork my repo.
- [ ] Add @rusher81572 as a contributor on npm, not sure how I can do that but that will make sure I can't just run off with the module and end up with a situation where nobody is maintaining it and nobody can push a new version to npm.

**Comment by TokinRing** [comment]
Why not put picluster-getos up on the PiCluster GitHub? Then it would be a simple matter of:
```
"getos": "git+https://github.com/picluster/picluster-getos.git"
```

PiCluster doesn't really have an npm presence afaik, so no need to muck with that now.

This one should be simple, so lets see this one closed here soon.

**Comment by TokinRing** [comment]
Resolved in commit 78b0a92
